src/odemis/acq/align/autofocus.py

```diff
@@ -114,6 +115,21 @@ def MeasureOpticalFocus(image):
     return cv2.Laplacian(image, cv2.CV_64F).var()
 
 
+def Measure1d(image):
+    def gauss(x, *p):
+        a, b, c, d = p
+        y = a * numpy.exp(-numpy.power((x - b), 2.) / (2. * c ** 2.)) + d
+        return y
+
+    signal = numpy.squeeze(image)
```

Comment:
It'd be nice to explain the rough idea of the algorithm. It fits a Gaussian? And then report the inverse of the gaussian width?